Iterators and Closures/Standard Library Types/Intro/task.md

```diff
@@ -276,10 +284,4 @@ For example, if for some reason we wanted to take the values produced by an inst
 
 Note that `zip` produces only four pairs; the theoretical fifth pair `(5, None)` is never produced because `zip` returns `None` when either of its input iterators return `None`.
 
-All of these method calls are possible because we specified how the `next` method works, and the standard library provides default implementations for other methods that call `next`.
-
-For the Arc exercise check out the chapter [Shared-State Concurrency](https://doc.rust-lang.org/book/2018-edition/ch16-03-shared-state.html) of the Rust Book.
-
-For the Iterator exercise check out the chapters [Iterator](https://doc.rust-lang.org/book/2018-edition/ch13-02-iterators.html) of the Rust Book and the [Iterator documentation](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.htmlj).
-
-Do not adjust your monitors -- iterators 1 and 2 are indeed missing. Iterator 3 is a bit challenging so we're leaving space for some exercises to lead up to it!
+All of these method calls are possible because we specified how the `next` method works, and the standard library provides default implementations for other methods that call `next`.
```

Comment:
All of these method calls are possible because we've specified how the `next` method works and the standard library provides default implementations for other methods that call `next`.